docs/customer-swagger.yaml

```diff
@@ -828,28 +841,9 @@ components:
         type: CustomerList
         customers:
           - id: 'https://api.test.nva.unit.no/customers/3fa85f64-5717-4562-b3fc-2c963f66afa6'
-            identifier: '3fa85f64-5717-4562-b3fc-2c963f66afa6'
-            type: Customer
             createdDate: '2021-05-07T14:29:04.144Z'
-            modifiedDate: '2021-05-07T14:29:04.144Z'
-            name: 'Universitet for norsk Reindrift'
             displayName: 'Universitet for Reindrift'
-            shortName: 'UNIREIN'
-            archiveName: 'unirein'
-            cname: 'arkiv.unirein.no'
-            institutionDns: 'unirein.no'
-            feideOrganizationId: '111111111'
-            cristinId: 'https://api.cristin.no/v2/institutions/11110'
-            vocabularies:
-              - id: 'https://example.org/example#'
-                type: 'Vocabulary'
-                name: 'Example VOC'
-                active: 'default'
```

Comment:
Is all this some old swagger specs?

Reply:
I don't think this swagger documentation matched the output from that endpoint even before I started working this branch.